spec.html

```diff
@@ -11982,12 +11982,12 @@ <h1>InitializeHostDefinedRealm ( ): either a normal completion containing ~unuse
   <emu-clause id="sec-agents">
     <h1>Agents</h1>
 
-    <p>An <dfn id="agent" variants="agents">agent</dfn> comprises a set of ECMAScript execution contexts, an execution context stack, a running execution context, an <dfn id="agent-record" variants="Agent Records">Agent Record</dfn>, and an <dfn id="executing-thread" variants="executing threads">executing thread</dfn>. Except for the executing thread, the constituents of an agent belong exclusively to that agent.</p>
-    <p>An agent's executing thread executes a job on the agent's execution contexts independently of other agents, except that an executing thread may be used as the executing thread by multiple agents, provided none of the agents sharing the thread have an Agent Record whose [[CanBlock]] field is *true*.</p>
+    <p>An <dfn id="agent" variants="agents">agent</dfn> comprises a set of ECMAScript execution contexts, an execution context stack, a running execution context, an <dfn id="agent-record" variants="Agent Records">Agent Record</dfn>, and an <dfn id="executing-thread" variants="executing threads">executing thread</dfn>. Except for the executing thread, Shared Data Blocks, and specification values related to Shared Data Blocks, the constituents of an agent belong exclusively to that agent.</p>
```

Comment:
This insertion is odd. Shared Data Blocks etc *weren't* just listed as constituents of an agent, so it doesn't make sense to say that they're among the constituents that aren't exclusive to the agent.

I think it would be more helpful to expand the Agent Clusters section to say that every Shared Data Block etc 'belongs to' an agent cluster. (Or, an agent cluster comprises zero or more Shared Data Blocks, WaiterList Records, Waiter Records, whatever.)

Reply:
Good suggestion, I've added a list of spec values to the agent cluster section.